test/functional/sqlsrv/sqlsrv_ae_insert_sqltype_binary_size.phpt

```diff
@@ -0,0 +1,271 @@
+--TEST--
+Test for inserting encrypted data into binary types columns with different sizes
+--DESCRIPTION--
+Test implicit conversions between different binary types of different sizes
+With Always Encrypted, implicit conversion works if:
+1. From input of SQLSRV_SQLTYPE_BINARY(n) to a larger binary(m) column where n <= m
+2. From input of SQLSRV_SQLTYPE_BINARY(n) to a larger varbinary(m) column where n <= m (m can be max)
+3. From input of SQLSRV_SQLTYPE_VARBINARY(n) to a larger binary(m) column where n <= m
+4. From input of SQLSRV_SQLTYPE_VARBINARY(n) to a larger varbinary(m) column where n <= m (m can be max)
+Without AlwaysEncrypted, implicit conversion between different binary types and sizes works
+--SKIPIF--
```

Comment:
why are <= sign highlighted, do you know?

Reply:
no idea, but I don't think it would affect anything